game-core/src/main/java/games/strategy/net/ClientMessenger.java

```diff
@@ -83,9 +84,7 @@ public ClientMessenger(
           }
         } catch (final ConnectException e) {
           throw new RuntimeException(
-              String.format(
-                  "Could not connect host: %s, port: %s, name: %s, mac: %s", host, port, name, mac),
-              e);
+              String.format("Could not connect host: %s, port: %s", host, port), e);
```

Comment:
name and mac are removed from this message, it's not yet used and not part of trying to connect to a remote host. Instead of updating the 'mac' name here, it's just removed.